src/Core/Settings.h

```diff
@@ -1046,6 +1046,7 @@ class IColumn;
     M(Bool, input_format_orc_allow_missing_columns, true, "Allow missing columns while reading ORC input formats", 0) \
     M(Bool, input_format_orc_use_fast_decoder, true, "Use a faster ORC decoder implementation.", 0) \
     M(Bool, input_format_orc_filter_push_down, true, "When reading ORC files, skip whole stripes or row groups based on the WHERE/PREWHERE expressions, min/max statistics or bloom filter in the ORC metadata.", 0) \
+    M(Bool, input_format_orc_read_use_writer_time_zone, false, "Whether use the writer's time zone in ORC stripe for ORC row reader, the default ORC row reader's time zone is GMT.", 0) \
```

Comment:
GMT ? Or UTC ? GMT!=UTC

Reply:
It is `GMT`, and this is defined at `orc/c++/src/Options.hh`, in `struct RowReaderOptionsPrivate`.

Reply:
It is `GMT`, and this is defined at `orc/c++/src/Options.hh`, in `struct RowReaderOptionsPrivate`. @den-crane